Patch 2: Fix

/builddir/build/BUILD/VirtualBox-6.1.4/src/VBox/Debugger/DBGCDumpImage.cpp: In function 'const char* dbgcMachoCpuType(uint32_t, uint32_t)':
/builddir/build/BUILD/VirtualBox-6.1.4/include/iprt/formats/mach-o.h:41:46: error: narrowing conversion of '-1' from 'int' to 'unsigned int' [-Wnarrowing]
   41 | #define CPU_TYPE_ANY                INT32_C(-1)
      |                                              ^
/builddir/build/BUILD/VirtualBox-6.1.4/src/VBox/Debugger/DBGCDumpImage.cpp:471:14: note: in expansion of macro 'CPU_TYPE_ANY'
  471 |         case CPU_TYPE_ANY:          return "CPU_TYPE_ANY";
      |              ^~~~~~~~~~~~

--- VirtualBox-6.1.6/src/VBox/Debugger/DBGCDumpImage.cpp.orig	2020-04-09 20:37:26.000000000 +0300
+++ VirtualBox-6.1.6/src/VBox/Debugger/DBGCDumpImage.cpp	2020-05-08 01:32:01.400447560 +0300
@@ -468,7 +468,7 @@ static const char *dbgcMachoCpuType(uint
 {
     switch (iType)
     {
-        case CPU_TYPE_ANY:          return "CPU_TYPE_ANY";
+        default:                    return "CPU_TYPE_ANY";
         case CPU_TYPE_VAX:          return "VAX";
         case CPU_TYPE_MC680x0:      return "MC680x0";
         case CPU_TYPE_X86:          return "X86";
